lib/mix/tasks/triplex.init.tenant.ex

```diff
@@ -0,0 +1,58 @@
+defmodule Mix.Tasks.Triplex.Init.Tenant do
```

Comment:
What about renaming this task to `triplex.mysql.install`?

Reply:
sure thing